### DISCUSSION TITLE
241125_월

### DISCUSSION THREAD

**Discussion by minrongtic** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#50] [TIL]
**TO DO** 

- [x] 노션에 Optimizer 정리

**Comment by Yoon0717** [comment]
- [x] optimizer, scheduler, seed 실험
- [x] EDA 한번 더

**Reply by minrongtic** [reply]
seed 21로도 해봐줄 수 있나요

**Reply by Yoon0717** [reply]
팅부동

**Reply by Soy17** [reply]
팅부동

**Comment by Soy17** [comment]
- [ ]  augmentation 실험 진짜로 진행
- [ ] 2048 해보기
- [ ] 강의/퀴즈 마무리 ~~

**Comment by sejongmin** [comment]
- [x] psuedo labeling 진행해보기
- [x] sam2unet 해상도 최대한 키워보기
- [x] sam2unet augmentation 실험
- [ ] sam2unet resume 코드 작성해보기

**Comment by june21a** [comment]
- [x] 2048로 돌아가는 모델 찾아서 돌려보기
- [x] 실험 내용 앙상블 포함해서 모두 정리하기

**Reply by minrongtic** [reply]
Unet++ efficientnet b0 <<<< 확실히 됨. 배치사이즈 2 
Unet++ efficientnet b1, b2, b3 <<<< 될까?

**Reply by june21a** [reply]
흠.... 테스트 해보는걸루!

**Comment by wonjeongjeong** [comment]
- [ ] DeepLabV3+ moblienet Sharpening/CHALE Augmentation 실험
- [x] 5~7 퀴즈 풀기
